src/store/setToolMode.js

```diff
@@ -56,12 +63,39 @@ const setToolActiveForElement = function(
         options[`is${interactionType}Active`] = false;
       }
     });
+
+    if (
+      globalConfiguration.state.showSVGCursors &&
+      tool.supportedInteractionTypes.includes('Mouse')
+    ) {
+      _setToolCursorIfPrimary(element, options, tool);
+    }
   }
 
   // Resume normal behavior
   setToolModeForElement('active', null, element, toolName, options);
 };
 
+function _setToolCursorIfPrimary(element, options, tool) {
+  let mouseButtonMask;
+
+  if (typeof options === 'number') {
+    mouseButtonMask = options;
```

Comment:
```suggestion
    mouseButtonMask = [options];
```
You're expecting `mouseButtonMask` to be an array on line 88.

Reply:
Good catch.